python/kserve/kserve/models/v1beta1_oauth_config.py

```
@@ -51,18 +51,20 @@ class V1beta1OauthConfig(object):
     attribute_map = {
         'cpu_limit': 'cpuLimit',
         'cpu_request': 'cpuRequest',
         'image': 'image',
         'memory_limit': 'memoryLimit',
-        'memory_request': 'memoryRequest'
+        'memory_request': 'memoryRequest',
+        'timeout_seconds': 'timeoutSeconds'
     }
 
-    def __init__(self, cpu_limit='', cpu_request='', image='', memory_limit='', memory_request='', local_vars_configuration=None):  # noqa: E501
+    def __init__(self, cpu_limit='', cpu_request='', image='', memory_limit='', memory_request='', timeout_seconds=None, local_vars_configuration=None):  # noqa: E501
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)